CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
```

Comment:
```suggestion
```

The `keepachangelog` format seems to be somewhat misleading as I am more leaning towards the format of the Elixir changelog format using "Enhancements" and "Bug fixes" instead of "Added" or "Fixed".